### ISSUE TITLE
Dedicated Server Crash on Augmented Sextant Use

### ISSUE THREAD

**Issue by Aiva418** [HellFirePvP/AstralSorcery#1263]
I am using the latest version of Astral Sorcery as of writing this (astralsorcery-1.12.2-1.10.23.jar), as well as over 100 other mods (which should be listed in both crash reports). I crafted an augmented sextant and opened the sextant interface. Opening and closing the interface does not appear to cause issues, as I was able to open another gui afterwards. The server seems to hang when matching a location to its corresponding star, as no other tileentities are interactable upon exiting the sextant view afterwards. I was able to replicate this crash on my server by using the sextant and matching a location to a star, after which I timed out and the server crashed. I did not experience this issue in singleplayer.

[crash-2020-04-13_23.14.12-server.txt](https://github.com/HellFirePvP/AstralSorcery/files/4473220/crash-2020-04-13_23.14.12-server.txt)

[crash-2020-04-13_23.41.07-server.txt](https://github.com/HellFirePvP/AstralSorcery/files/4473217/crash-2020-04-13_23.41.07-server.txt)

[latest.log](https://github.com/HellFirePvP/AstralSorcery/files/4473236/latest.log)

If you need more information, let me know.

**Comment by Doomgull** [comment]
Server load is too high, and server tick rate is exceeding the forge watchdog: 

[23:39:29] [Server thread/WARN] [net.minecraft.server.MinecraftServer]: Can't keep up! Did the system time change, or is the server overloaded? Running 28563ms behind, skipping 571 tick(s)
[23:40:07] [Server thread/WARN] [net.minecraft.server.MinecraftServer]: Can't keep up! Did the system time change, or is the server overloaded? Running 15111ms behind, skipping 302 tick(s)
[23:41:07] [Server Watchdog/FATAL] [net.minecraft.server.dedicated.ServerHangWatchdog]: A single server tick took 60.05 seconds (should be max 0.05)
[23:41:07] [Server Watchdog/FATAL] [net.minecraft.server.dedicated.ServerHangWatchdog]: Considering it to be crashed, server will forcibly shutdown.
[23:41:07] [Server Watchdog/WARN] [org.dimdev.utils.ModIdentifier]: Failed to identify hellfirepvp.astralsorcery.common.network.packet.client.PktRequestSextantTarget$$Lambda$3653.00000000BC07C2B0 (untransformed name: hellfirepvp.astralsorcery.common.network.packet.client.PktRequestSextantTarget$$Lambda$3653.00000000BC07C2B0)

What is different server vs singleplayer with your hosting? Memory availability? CPU? What are you matching in these tests, biome or building? Is it a vanilla biome? How many biomes are in your world?

**Comment by Aiva418** [comment]
I get the same crash for both biomes and buildings.

The server is running on the same CPU as what I have on my laptop, though I've allocated just over 5GB RAM on my client and the server only has 4. And the only difference I can see in software is that the server isn't running any purely clientside mods, and that it probably isn't running on MacOS (I'm not sure what the host uses).

I just remembered that I also used a different world type when testing in singleplayer. I have the mod Climatic Biomes installed, and I'm using its world generator settings. The server also has Biomes O' Plenty, so there's quite a lot of biomes in the world.

Having just tested again in singleplayer, I was able to achieve a crash by roughly the same process, but it required some extra clicking in the sextant menu and resulted in a clientside error rather than a serverside one. It definitely isn't the same issue, as the server one happened roughly a minute after every time I made a match, and that didn't happen in singleplayer. Still, I'll provide the crash log in case it's related:

[crash-2020-04-15_19.34.21-client.txt](https://github.com/HellFirePvP/AstralSorcery/files/4484364/crash-2020-04-15_19.34.21-client.txt)
